src/explore.ts

```diff
@@ -85,10 +85,45 @@ async function loadSourceMap(codeFile: File, sourceMapFile?: File): Promise<Sour
   };
 }
 
+function detectEOL(content: string): string {
+  const lineBreakIndex = content.indexOf('\n', 1);
+
+  return content[lineBreakIndex - 1] === '\r' ? '\r\n' : '\n';
+}
+
+interface ComputeFileSizesContext {
+  generatedLine: number;
+  generatedColumn: number;
+  line: string;
+  eol: string;
+}
+
+function checkInvalidMappingColumn({
+  generatedLine,
+  generatedColumn,
+  line,
+  eol,
+}: ComputeFileSizesContext): void {
+  const maxColumnIndex = line.length - 1;
+
+  // Ignore case when source map references EOL character (e.g. when generated by typescript)
```

Comment:
TypeScript (capitalization). Maybe include a reference to the relevant issue for context.